app/controllers/kondos_controller.rb

```diff
@@ -1,6 +1,6 @@
 class KondosController < ApplicationController
   def index
-    @kondos = Kondo.order(created_at: :desc)
+    authorize @kondos = Kondo.order(created_at: :desc)
```

Comment:
Maybe it works this way but might be safer to follow the class example and use @kondos = policy_scope(Kondo).order(created_at: :desc) :)

Reply:
but yeah if you skip the pundit for index we probably don't need to use policy_scope at all :) In that case I would maybe not use authorize either actually... my knowledge of pundit is way too limited haha